Source/ARTRealtimePresence.m

```diff
@@ -15,26 +15,65 @@
 #import "ARTStatus.h"
 #import "ARTPresence+Private.h"
 
+#pragma mark - ARTRealtimePresenceQuery
+
+@implementation ARTRealtimePresenceQuery
+
+- (instancetype)initWithLimit:(NSUInteger)limit clientId:(NSString *)clientId connectionId:(NSString *)connectionId {
+    self = [super initWithLimit:limit clientId:clientId connectionId:connectionId];
+    if (self) {
+        _waitForSync = true;
+    }
+    return self;
+}
+
+@end
+
+#pragma mark - ARTRealtimePresence
+
+@interface ARTRealtimePresence ()
+
+@property (readonly, getter=getChannel) ARTRealtimeChannel *channel;
+
+@end
+
 @implementation ARTRealtimePresence
 
 - (instancetype)initWithChannel:(ARTRealtimeChannel *)channel {
-    return [super initWithChannel:channel];
+    self = [super initWithChannel:channel];
+    return self;
 }
 
-- (ARTRealtimeChannel *)channel {
+- (ARTRealtimeChannel *)getChannel {
     return (ARTRealtimeChannel *)super.channel;
 }
 
-- (void)get:(ARTRealtimePresenceQuery *)query callback:(void (^)(ARTPaginatedResult<ARTPresenceMessage *> * _Nullable, NSError * _Nullable))callback {
-    [[self channel] throwOnDisconnectedOrFailed];
-    [super get:query callback:callback];
+- (void)get:(void (^)(NSArray<ARTPresenceMessage *> *, ARTErrorInfo *))callback {
+    [self get:[[ARTRealtimePresenceQuery alloc] init] callback:callback];
+}
+
+- (void)get:(ARTRealtimePresenceQuery *)query callback:(void (^)(NSArray<ARTPresenceMessage *> *, ARTErrorInfo *))callback {
+    [self.channel throwOnDisconnectedOrFailed];
+    [self.channel attach:^(ARTErrorInfo *error) {
+        if (query.waitForSync) {
```

Comment:
What about the rest of the members in ARTRealtimePresenceQuery?

By the way, according to [the spec](http://docs.ably.io/client-lib-development-guide/features/#idl), the query parameters for this method are `waitForSync`, `clientId` and `connectionId`. But our ARTRealtimePresenceQuery inherits from REST's ARTPresenceQuery, which has other parameters.